packages/SwingSet/src/liveslots/collectionManager.js

```diff
@@ -71,32 +70,9 @@ function matchAny(patt) {
 }
 
 function throwNotDurable(value, slotIndex, serializedValue) {
-  const body = JSON.parse(serializedValue.body);
-  let encodedValue;
-  try {
-    encodedValue = decodeToJustin(
-      {
-        body,
-        slots: serializedValue.slots,
-      },
-      true,
-    );
-  } catch (justinError) {
-    const err = assert.error(
-      X`value is not durable: ${value} at slot ${q(
-        slotIndex,
-      )} of ${serializedValue}`,
-    );
-    assert.note(
-      err,
-      X`decodeToJustin(${serializedValue}) threw ${justinError} `,
-    );
-    throw err;
-  }
   assert.fail(
-    X`value is not durable: ${value} at slot ${q(
-      slotIndex,
-    )} of ${encodedValue}`,
+    // prettier-ignore
+    X`value is not durable: ${value} at slot ${q(slotIndex)} of ${serializedValue.body}`,
```

Comment:
Do we still have need of ```assert.fail(X`…`)```, or can it now be ``` Fail`…` ```?

Reply:
I didn't know we could do that, but I don't (yet) find `Fail..` to be more readable than `assert.fail(..)`.

Reply:
It was introduced in https://github.com/endojs/endo/pull/1334 .